sky/server/server.py

```diff
@@ -178,6 +225,16 @@ async def dispatch(self, request: fastapi.Request, call_next):
 app.add_middleware(RequestIDMiddleware)
 app.include_router(jobs_rest.router, prefix='/jobs', tags=['jobs'])
 app.include_router(serve_rest.router, prefix='/serve', tags=['serve'])
+# Serve static files from the dashboard directory
+if os.path.exists(server_constants.DASHBOARD_DIR):
+    app.mount('/dashboard/_next',
+              staticfiles.StaticFiles(
+                  directory=f'{server_constants.DASHBOARD_DIR}/_next'),
+              name='dashboard')
+else:
+    logger.warning(
+        f'Dashboard directory {server_constants.DASHBOARD_DIR} does not exist.'
+        f' Skipping dashboard static files mount.')
```

Comment:
I assume this is for testing, and we will remove it if we decide to stick with the manually serving method. Is that right?

Reply:
Change to create `out` dir if it does not exist.